contracts/modules/v2/StakingPool.sol

```diff
@@ -448,8 +457,7 @@ contract StakingPool is IStakingPool, ERC721 {
 
       uint withdrawable = stakeToWithdraw + rewardsToWithdraw;
 
-      // TODO: use TC instead
-      nxm.transfer(ownerOf(tokenId), withdrawable);
+      tokenController.withdrawStakedNXM(ownerOf(tokenId), withdrawable, poolId);
```

Comment:
`withadrawable` includes stake (deposits) and rewards but the function handles only deposits. Let's make it accept both.